eng/docs.targets

```diff
@@ -39,10 +30,13 @@
       <DocFile>
         <SubFolder>%(Culture)/</SubFolder>
       </DocFile>
-      <!-- set item expected by depproj -->
-      <NuGetDeploy Include="@(DocFile)" />
     </ItemGroup>
 
     <Error Condition="'%(DocFile.Culture)' == 'unknown'" Text="Unknown language folder '%(LCID)' for doc files '@(DocFile)'" />
+    
+    <Copy SourceFiles="@(DocFile)"
```

Comment:
Here and all the other places you've added calls to copy, consider including the other parameters like `UseHardlinksIfPossible`, `Retries`, etc.